tests/test_datastructures.py

```diff
@@ -0,0 +1,37 @@
+from cms.test_utils.testcases import CMSTestCase
+
+from djangocms_versioning.datastructures import VersionableItem
+from djangocms_versioning.test_utils.factories import (
+    AnswerFactory,
+    PollVersionFactory,
+)
+from djangocms_versioning.test_utils.polls.models import PollContent
+
+
+class VersionableItemTestCase(CMSTestCase):
+
+    def setUp(self):
+        self.initial_version = PollVersionFactory()
+        AnswerFactory.create_batch(
```

Comment:
Do you really need answers in these tests? I think they're needed for the copy tests, but not sure they add anything here?